docs/manual/getting_started.rst

```diff
@@ -173,21 +221,21 @@ This step will copies everything to:
 At this point, you are ready to use Eclipse Cyclone DDS in your
 projects.
 
-**Note:** The default build type is a release build with debug
-information included (RelWithDebInfo). This is a convenient type of
-build to use from applications because of a good mix between performance
-and still being able to debug things. If you'd rather have a Debug or
-pure Release build, set ``CMAKE_BUILD_TYPE`` accordingly.
+**Note:** The default build type is a release build that includes debugging 
+information (RelWithDebInfo). 
+
+This build is commonly used from applications because of its high performance and debugging capabilities.  
+If you'd rather have a Debug or pure Release build, set ``CMAKE_BUILD_TYPE`` accordingly.
 
 If you want to contribute to Cyclone DDS, please refer to `Appendix I
 Contributing to Eclipse Cyclone DDS for
 Windows. <#appendix-i-contributing-to-eclipse-cyclone-dds>`__
 
 Installation with product installer for Windows
```

Comment:
Yeah clearly I did not review this properly last time around: there is no product installer included in the open source releases (although there are tools to build one). I think it is again best to remove this section. Also, the image naming sceme is not the best, renaming the images and updating the references is not a bad idea either (although tedious I agree)